.evergreen/run-mongodb-aws-ecs-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Don't trace since the URI contains a password that shouldn't show up in the logs
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       MONGODB_URI             Set the URI, including an optional username/password to use to connect to the server via MONGODB-AWS
+#                               authentication mechanism
+#       JDK                     Set the version of java to be used.  Java versions can be set from the java toolchain /opt/java
+#                               "jdk5", "jdk6", "jdk7", "jdk8", "jdk9", "jdk11"
+
+############################################
+#            Main Program                  #
+############################################
+
+echo "Running MONGODB-AWS ECS authentication tests"
+
+if ! which java ; then
+    echo "Installing java..."
+    apt install openjdk-11-jdk -y
+fi
+
+if ! which git ; then
+    echo "installing git..."
+    apt install git -y
+fi
+
+MONGODB_URI="mongodb://localhost/aws?authMechanism=MONGODB-AWS&authSource=\$external"
```

Comment:
Ah, here too.  Change to:

    mongodb://localhost/?authMechanism=MONGODB-AWS

Reply:
Saw your comment about the test using the database name from the URI.  That seems fine then.

Reply:
Ack